CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+2.0.1 / 2017-03-10
```

Comment:
Good spot. I have updated this and updated the changelog with the new dependencies.